eclair-core/src/main/scala/fr/acinq/eclair/db/DbEventHandler.scala

```diff
@@ -116,6 +118,11 @@ class DbEventHandler(nodeParams: NodeParams) extends Actor with ActorLogging {
       auditDb.add(ChannelEvent(e.channelId, e.commitments.remoteParams.nodeId, e.commitments.commitInput.txOut.amount, e.commitments.localParams.isFunder, !e.commitments.announceChannel, event))
       channelsDb.updateChannelMeta(e.channelId, event)
 
+    case u: LocalChannelUpdate =>
```

Comment:
NB: this will trigger a lot of noise, we resend this every time a peer reconnects...but maybe it's ok since it doesn't contain that much data